tests/Main/js/1foo.js

```diff
@@ -1,42 +1,46 @@
 "use strict";
 
-module.exports = {
 
-    foo: "foo",
+export let foo = "foo";
 
-    apply: function(f, x, y) {
-        return f(x, y);
-    },
+export function apply(f, x, y) {
+  return f(x, y);
+};
 
-    square: function(x) {
-        return x * x;
-    },
+export function square(x) {
+  return x * x;
+};
 
-    add: function(x, y) {
-        return x + y;
-    },
+export function add(x, y) {
+  return x + y;
+};
 
-    MyClass: class {
-        constructor(v) {
-            this.__value = typeof v === "string" ? v : "haha";
-        }
+export class MyClass {
+  constructor(v) {
+    this.__value = typeof v === "string" ? v : "haha";
+  }
 
-        get value() {
-            return this.__value;
-        }
+  get value() {
+    return this.__value;
+  }
 
-        static foo(i) {
-            return typeof i === "number" ? i * i : "foo";
-        }
+  static foo(i) {
+    return typeof i === "number" ? i * i : "foo";
+  }
 
-        bar(s) {
-            return typeof s === "string" ? s.toUpperCase() : "bar";
-        }
-    },
+  bar(s) {
+    return typeof s === "string" ? s.toUpperCase() : "bar";
+  }
+};
 
-    fooOptional: {
-        Foo1() { return arguments.length },
-        Foo2() { return arguments.length },
-        Foo3() { return arguments.length }
-    }
+export const fooOptional = {
+  Foo1() {
+    return arguments.length;
+  },
+  Foo2() {
+    return arguments.length;
+  },
+  Foo3() {
+    return arguments.length;
+  }
```

Comment:
Please see the comment below about commonjs modules. If possible, I'd prefer to keep them to check importing from commonjs modules also works. There are already other JS files in the test project with ES2015 modules.